.travis.yml

```diff
@@ -12,7 +12,6 @@ install:
   - pip install -U pip wheel
   - pip install -r test_requirements.txt
   - pip install -r requirements.txt
-  - pip install https://github.com/hasgeek/coaster/archive/master.zip
```

Comment:
Doesn't this work if it's in `requirements.txt`?

Reply:
It should. I'm not sure why it was added here.